modules/entity_version_history/tests/modules/entity_version_history_test/src/Entity/HistoryEntityTest.php

```diff
@@ -18,12 +18,20 @@
  *   id = "history_entity_test",
```

Comment:
So actually, we don't technically need to ship with this new test entity type as we can/should use one of the core EntityTest ones and add there version fields programatically. Can you check if possible and quickly replace? See the `entity_test` module from core.